index.js

```diff
@@ -31,6 +32,11 @@ class WebpackReactComponentNamePlugin {
       }
     )
 
```

Comment:
I've never worked with a webpack plugin before, so this may just be magic, but I don't see this value used anywhere. Is it necessary?

Reply:
Good catch - this was originally in here for debugging purposes but I've since removed references to it. Fixed